Gemfile

```diff
@@ -3,4 +3,4 @@ source "https://rubygems.org"
 gemspec
 
 # fixes 2.3.0 ffi bundle error
```

Comment:
I wish we could deprecate comments. This seems like it is nearing its sell-by date.